frame/bounties/src/lib.rs

```diff
@@ -440,15 +504,23 @@ decl_module! {
 								}
 							},
 						}
+
+						// Unreserve the fee for Curator which got reserved during "accept_curator()"
+						let bounty_account = Self::bounty_account_id(bounty_id);
+						T::Currency::unreserve(&bounty_account, bounty.fee);
```

Comment:
what is this?

Reply:
Took reducing-balance approach, as design choice for managing the funds for subbounties.

In below sequence flow graph, 
tried to capture operation on funds 
for different extrincis.

![](https://i.imgur.com/FrLZfOS.jpg)

[ Link to the image | https://i.imgur.com/FrLZfOS.jpg ]

In this context Sequence no 60, 61 ("unassign_curator()"), 
curator fee gets unreserved, which gets reserved during "accept_curator()"
sequence no 4, 5.

![](https://i.imgur.com/nQH8cEM.png)

![](https://i.imgur.com/y9xwDxh.png)

Reply:
Design for fund management got changed, As suggested now each subbounty has its own unique account.